sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -547,6 +547,22 @@ protected EventProcessorClient()
         /// <exception cref="InvalidOperationException">Occurs when this method is invoked without <see cref="ProcessEventAsync" /> or <see cref="ProcessErrorAsync" /> set.</exception>
         ///
         public virtual async Task StartProcessingAsync(CancellationToken cancellationToken = default)
+            => await StartProcessingImplAsync(true, cancellationToken).ConfigureAwait(false);
+
+        /// <summary>
+        ///   Signals the <see cref="EventProcessorClient" /> to begin processing events.  Should this method be called while the processor
+        ///   is running, no action is taken.
+        /// </summary>
+        ///
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> instance to signal the request to cancel the start operation.  This won't affect the <see cref="EventProcessorClient" /> once it starts running.</param>
+        ///
+        /// <exception cref="EventHubsException">Occurs when this <see cref="EventProcessorClient" /> instance is already closed.</exception>
+        /// <exception cref="InvalidOperationException">Occurs when this method is invoked without <see cref="ProcessEventAsync" /> or <see cref="ProcessErrorAsync" /> set.</exception>
+        ///
+        public virtual void StartProcessing(CancellationToken cancellationToken = default)
+            => StartProcessingImplAsync(false, cancellationToken).EnsureCompleted();
+
+        private async Task StartProcessingImplAsync(bool async, CancellationToken cancellationToken)
```

Comment:
Please add doc comments;  Event Hubs uses them for all members, including non-public.

Reply:
Fixed.